libfwupdplugin/fu-device-event.h

```diff
@@ -11,6 +11,8 @@
 #define FU_TYPE_DEVICE_EVENT (fu_device_event_get_type())
 G_DECLARE_FINAL_TYPE(FuDeviceEvent, fu_device_event, FU, DEVICE_EVENT, GObject)
 
+FuDeviceEvent *
+fu_device_event_new(const gchar *id);
```

Comment:
can you split this commit out to a new PR pls

Reply:
done: https://github.com/fwupd/fwupd/pull/9114